build.rs

```diff
@@ -1,23 +1,23 @@
-extern crate version_check as rustc;
+extern crate autocfg;
 
 fn main() {
-    if is_rustc_at_least("1.5.0") {
-        println!("cargo:rustc-cfg=__unicase__iter_cmp");
-    }
+    autocfg::rerun_path(file!());
+
+    let ac = autocfg::new();
 
-    if is_rustc_at_least("1.13.0") {
-        println!("cargo:rustc-cfg=__unicase__default_hasher");
+    if ac.probe_rustc_version(1, 5) {
```

Comment:
Rather than probing for versions, it’s better to probe for specific features, using something like https://docs.rs/autocfg/0.1.7/autocfg/struct.AutoCfg.html#method.emit_has_path

Reply:
So does the existing `__unicase__iter_cmp` feature correspond to `ac.emit_has_path("core::iter::Iterator::cmp")`?